libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -570,6 +700,10 @@ private void EnsureDependenciesInstalled()
                 {
                     installedDependencies = true;
 
+                    var id = 0u;
+                    var noActivity = 0;
+                    var activity = 1000;
+                    var input = 2000;
```

Comment:
Use static int as configuration somewhere?

Reply:
This has been changed to not do this any more.